pkg/kubelet/volumes.go

```diff
@@ -66,10 +64,16 @@ func (vh *volumeHost) NewWrapperMounter(volName string, spec volume.Spec, pod *a
 	}
 
 	b, err := vh.kubelet.newVolumeMounterFromPlugins(&spec, pod, opts)
```

Comment:
`newVolumeMounterFromPlugins` should return an error if there's no suitable volume plugin.